src/MEDS_tabular_automl/scripts/launch_autogluon.py

```
@@ -0,0 +1,81 @@
+from importlib.resources import files
+from pathlib import Path
+
+import hydra
+import pandas as pd
+from loguru import logger
+from omegaconf import DictConfig
+
+from MEDS_tabular_automl.dense_iterator import DenseIterator
+
+from ..utils import hydra_loguru_init
+
+config_yaml = files("MEDS_tabular_automl").joinpath("configs/launch_autogluon.yaml")
+if not config_yaml.is_file():
+    raise FileNotFoundError("Core configuration not successfully installed!")
+
+
+@hydra.main(version_base=None, config_path=str(config_yaml.parent.resolve()), config_name=config_yaml.stem)
+def main(cfg: DictConfig) -> float:
+    """Launches AutoGluon after collecting data based on the provided configuration.
+
+    Args:
+        cfg: The configuration dictionary specifying model and training parameters.
+    """
+
+    # print(OmegaConf.to_yaml(cfg))
+    if not cfg.loguru_init:
+        hydra_loguru_init()
+
+    # check that autogluon is installed
+    try:
+        import autogluon.tabular as ag
+    except ImportError:
+        logger.error("AutoGluon is not installed. Please install AutoGluon.")
```

do you want to raise an exception here? I don't think `logger.error` does necessarily.